worker/worker/worker.js

```diff
@@ -201,6 +201,9 @@ async function handler_witness_instagram_post(request) {
     }
 
     let kvObj = JSON.parse(kvEntry);
+    if (!kvObj?.link || !kvObj?.sig) {
```

Comment:
Can that happen?

Reply:
If given a non-existent handle,`{}` comes from the KV store.